app/db_utils.go

```diff
@@ -0,0 +1,160 @@
+package app
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
+	"log"
+	"strconv"
+	"strings"
+)
+
+func connectDB() (*sql.DB, error) {
+	var db *sql.DB
+	connectString := `
+		user=postgres 
+		password=meme 
+		host=35.192.115.150 
+		dbname=meme-db 
+		sslmode=disable 
+	`
+	db, openErr := sql.Open("postgres", connectString)
+	if openErr != nil {
+		log.Fatal(openErr)
+		return db, openErr
+	}
+
+	if err := db.Ping(); err != nil {
+		log.Fatal(err)
+		return db, err
+	}
+
+	return db, nil
+}
+
+func getMemeIdsByKeyword(db *sql.DB, input queryInput) ([]int, error) {
+	var memeIds []int
+
+	sqlQuery := fmt.Sprintf(
+		`
+		SELECT 
+			meme.id
+		FROM 
+			meme
+		WHERE 
+			meme.about LIKE '%%%s%%'
```

Comment:
What about queryInput.NumOfResult? Maybe you should add limit in sql query here.